p8e-migration/sql/V66__Add_UUID_Key_To_Affiliate.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE affiliate ADD COLUMN key_uuid UUID;
```

Comment:
Thought about making this a PK, might not be necessary, it adds more risk and effort as we have 6 tables that are FK to the public key.

Reply:
I totally forgot about signing and encryption keys being able to be different now. Shouldn't we add a signing_key_uuid and encryption_key_uuid here so they can be referenced independently?

Reply:
Right, was debating that I wanted to work on signing keys (the tougher of the 2 keys) first, get it working E2E, then start on encryption, think if I tried to tackle both of the keys at the same time, it might be very regression prone and confusing (like how the diff key PR was). When I get around to encryption I'll just PR a branch against the feature.

Reply:
That's a good plan. I just wanted to mention it as it was on my mind.